code-challenge/src/components/ArticleList/ArticleList.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './ArticleList.scss';
+import { List } from "@material-ui/core";
+import Article from "../Article/Article.lazy";
+import {RootStateOrAny, useSelector} from "react-redux";
+
+export interface ArticleListProps {
+    articles: any[]
+}
+
+const ArticleList = () => {
+    const { articles } = useSelector((state: RootStateOrAny) => state);
```

Comment:
This useSelector hook is pretty nice. Do you have any thoughts on using Context vs Redux?

Reply:
I wanted to use redux here just to show I could setup redux, but in an application this size, I would think of using context. With redux, I do also like using sagas and handling side effects in something not in charge of state. Is there a corollary to sagas in Context?

Reply:
Nah, not in context! For handling state and side effects, I use a combination of ReactQuery + Context.

Reply:
oooh shiny @ ReactQuery. Going to dig into this. Thanks for sharing!